### ISSUE TITLE
@Named not working

### ISSUE THREAD

**Issue by p500plus** [joinfaces/joinfaces#297]
Hello i am new to joinfaces but i realy like it already. Im trying to setup some basics project like spring boot + redis + spring security and primefaces. I have downloaded and try out your example project  (https://github.com/joinfaces/joinfaces-example). 
Inspired by this project i wanted to create my own project but as simple as possible with only necesary dependencies etc...

In my project i have only few dependencies
```xml
<dependencies>
    <dependency>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter</artifactId>
    </dependency>		

    <dependency>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-test</artifactId>
        <scope>test</scope>
    </dependency>
      
    <dependency>
        <groupId>org.joinfaces</groupId>
        <artifactId>jsf-spring-boot-starter</artifactId>
    </dependency>

    <dependency>
        <groupId>org.primefaces.extensions</groupId>
        <artifactId>all-themes</artifactId>
        <version>1.0.8</version>
    </dependency>  
</dependencies>
```
and a very small config

```properties
jsf.project-stage=Development
jsf.primefaces.theme=bootstrap
jsf.mojarra.preferXHTML=true
jsf.primefaces.font-awesome=true
server.port=80
```

then i have created simple TestBean with only one property
```java
@Named
@SessionScoped
public class TestBean implements Serializable {
	
	private String test;
	
	private static final long serialVersionUID = 1L;
	
	@PostConstruct
	public void init(){
		setTest("default value");
	}
	
	public String getTest(){
		return test;
	}
	
	public void setTest(String test){
		this.test = test;
	}
}
```

and then i have created index.xhtml
```
...
#{testBean.test}
...
```

when i run it like this no value was inserted. After a while i find out that when i change `@Named` into `@ManagedBean` everything work fine.

But i dont know why? Have i done something wrong? Am i missing any configuration or dependency? How its possiple that in your example project you are using `@Named` anotation and it works fine?

Thank you

**Comment by larsgrefer** [comment]
What happens if you use `@Component` instead of `@Named`?

**Comment by p500plus** [comment]
Well if i use  `@Component` it also doesnt work.

**Comment by larsgrefer** [comment]
In which package is your `TestBean` and in which package is your `@SpringBootApplication`?

Maybe the `TestBean` is not picked up by springs component-scan.

**Comment by p500plus** [comment]
I have uploaded my project to github so you can take a look
https://github.com/p500plus/simple_faces/tree/master

**Comment by larsgrefer** [comment]
**TL;DR**: Your `faces.TestBean` is not a spring bean, so it can't be resolved from your `index.html`. It's neither declared in any xml or java configuration nor picked up by any component scan.

Your [`@SpringBootApplication`](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-using-springbootapplication-annotation) annotated class is located in the `com.faces.experiment`-package. So spring boot's default component scan only picks up `@Named`, or `@Component` (meta-)annotated classes which are located in or below the `com.faces.experiment`-package.

I'd suggest you to read the documentation about [the code structure of a spring boot application](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-structuring-your-code) and expecially about [where to put the main application class](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-locating-the-main-class).

**Comment by p500plus** [comment]
Ouu such a silly mistake. Im sorry for your time. I already fix it.
Thank you very very much
